docs/source/flow_setup.rst

```diff
@@ -152,7 +150,7 @@ Getting started (rllab-multiagent)
 To run any of the RL examples, make sure to run
 ::
 
-    source activate flow
+    source activate flow-rllab
```

Comment:
This is not the case if you're not using rllab; you won't have this env.

Reply:
u create the environment during the rllab setup instructions

Reply:
this is in the Setting up section for rllab btw, which is totally optional and assumes u've installed rllab

Reply:
Mmm, I was confused by the git diff. Okay!